gef.py

```diff
@@ -5991,28 +5991,51 @@ class NopCommand(GenericCommand):
     aware."""
 
     _cmdline_ = "nop"
-    _syntax_  = ("{_cmdline_} [LOCATION] [--nb NUM_BYTES]"
```

Comment:
Can we maybe still support `--nb` (deprecate it) so we don't break users' scripts?

Reply:
no, because we are breaking all. Now the default behaviour its the correct .... So keep this arg == nothing

Reply:
Please, @hugsy take a look to our discord discussion and take a decission